Userland/Libraries/LibJS/Runtime/AbstractOperations.cpp

```diff
@@ -562,23 +648,47 @@ ThrowCompletionOr<Value> perform_eval(Value x, GlobalObject& caller_realm, Calle
     // 18. If runningContext is not already suspended, suspend runningContext.
     // FIXME: We don't have this concept yet.
 
+    // 19. Let evalContext be a new ECMAScript code execution context.
     ExecutionContext eval_context(vm.heap());
-    eval_context.realm = eval_realm;
+
+    // 20. Set evalContext's Function to null. (This was done in the construction of eval_context)
```

Comment:
I know you like this style of comment, but please start moving away from it - I have something in the works to automate diffing our spec comments with the actual spec, and that works best if comments are copied verbatim onto a single line :)

`// NOTE: This was done in the construction of eval_context.` on a new line would be preferable.

Reply:
Done :+1: